CHANGELOG-2.0.md

```diff
@@ -0,0 +1,16 @@
+# Changelog since v2.0.1
```

Comment:
We are currently keeping a different file per minor version.  Looks like we missed a changelog file for 2.0 itself.

Reply:
I've renamed the file.

We didn't have good changelog entries in the PRs leading up to v2.0.1, that probably explains it.